src/components/Canary/url.js

```diff
@@ -0,0 +1,24 @@
+const cleaner = (key, value) =>
+  // See re prototype pollution: https://book.hacktricks.xyz/pentesting-web/deserialization/nodejs-proto-prototype-pollution
+  key === "__proto__" || key === "constructor" ? undefined : value;
+
+export function encodeObjectToUrlSearchParams(object) {
```

Comment:
I think we are over encoding here, the parameters and value should be normal parameters, only the labels/filters  should be encoded as json